frontend/src/components/7.Result/ResultPage.js

```diff
@@ -64,6 +72,9 @@ function ResultPage(props) {
       });
   }, []);
 
+  // Creating the pie chart from the vote data.
+  // Each selected restaurant is shown as a legend and the overal voting 
```

Comment:
small nitpick but it should be 'overall'

Reply:
Thanks Revve, my bad I didnt catch that on the spell checking.